packages/gradle/src/utils/exec-gradle.ts

```diff
@@ -0,0 +1,60 @@
+import { workspaceRoot } from '@nx/devkit';
+import { ExecFileOptions } from 'child_process';
+import {
+  ExecFileSyncOptionsWithBufferEncoding,
+  execFile,
+  execFileSync,
+} from 'node:child_process';
+import { existsSync } from 'node:fs';
+import { join } from 'node:path';
+
+export function execGradle(
+  args: string[],
+  execOptions: ExecFileSyncOptionsWithBufferEncoding
+) {
+  const gradleBinaryPath = getGradleBinaryPath();
+
+  return execFileSync(gradleBinaryPath, args, execOptions);
+}
+
+export function getGradleBinaryPath() {
+  const gradleBinaryPath = join(workspaceRoot, './gradlew');
```

Comment:
Make sure this works on windows, I think there it should be `./gradlew.bat`?

Reply:
changed